bin/kmstool-enclave-cli/README.md

```diff
@@ -127,3 +127,17 @@ By doing that, this tool can be used by any programming langauge that can intera
    result_b64 = proc.communicate()[0].decode()
    plaintext_b64 = result_b64.split(":")[1].strip()
    ```
+
+## Troubleshooting
+
+### Missing Common CA Certificates
+If you are running `kmstool-enclave-cli` in an environment that does not have the common CA certificates installed, you will face the following error:
+```shell
+[ERROR] [2023-02-23T15:16:21Z] [00007efd15f94840] [tls-handler] - ctx: configuration error: Error initializing trust store (Error encountered in /tmp/crt-builder/s2n-tls/tls/s2n_x509_validator.c:120)
+[ERROR] [2023-02-23T15:16:21Z] [00007efd15f94840] [tls-handler] - Failed to set ca_path: (null) and ca_file (null)
+```
+
+To solve the problem, use an docker image that has common CA certificates pre-installed like `amazonlinux:2`. If you want to use a docker image with a smaller initial footprint, e.g. `debian:buster-slim`, you have to install the CA certificates during the docker build step similar to this:
```

Comment:
We're using a base scratch image for the final EIF and we explicitly [create](https://github.com/aws/aws-nitro-enclaves-sdk-c/blob/main/containers/Dockerfile.al2#L90) and copy everything that is minimally needed there. Might also worth mentioning.

Reply:
@alcioa thanks for pointing that out. Can you point me to the particular section in the build process? Will add a reference in the readme.

Reply:
@alcioa @petreeftime please review